yarn.lock

```diff
@@ -9224,6 +9237,16 @@ jss@*, jss@10.1.1, jss@^10.0.0, jss@^10.0.3:
     is-in-browser "^1.1.3"
     tiny-warning "^1.0.2"
 
+jss@10.2.0, jss@^10.0.3:
```

Comment:
There is a nonnegligible amount of dependencies duplication in this yarn.lock. While we have built-in support for solving this problem in the CI of the main repository, I think that for this time, https://github.com/atlassian/yarn-deduplicate#usage will be good enough.
You should see a diff like this: `1 file changed, 99 insertions(+), 1172 deletions(-)`.